src/breadcrumb-group/implementation.tsx

```diff
@@ -94,8 +111,21 @@ export function BreadcrumbGroupImplementation<T extends BreadcrumbGroupProps.Ite
   let breadcrumbItems = items.map((item, index) => {
     const isLast = index === items.length - 1;
 
+    const clickAnalyticsMetadata: GeneratedAnalyticsMetadataBreadcrumbGroupClick = {
+      action: 'click',
+      detail: {
+        position: `${index + 1}`,
+        label: `.${analyticsSelectors['breadcrumb-item']}`,
+        href: item.href || '',
+      },
+    };
+
     return (
-      <li className={styles.item} key={index}>
+      <li
+        className={styles.item}
+        key={index}
+        {...(isLast ? {} : getAnalyticsMetadataAttribute(clickAnalyticsMetadata))}
```

Comment:
It does not attach a listener, it just adds metadata to the field, so that they can be retrieved by an analytics engine. It is the responsibility of the analytics engine to attach the listeners.